client/account-settings/components/label-payment-method/style.scss

```diff
@@ -0,0 +1,38 @@
+.card.is-compact.label-payment-method {
```

Comment:
To bead specificity. The Calypso rules for `<Card>` include: `.card.is-compact {}`, so if we used `.card.label-payment-method {}` ours would have the same specificity and it would come down to which CSS file was included first. I prefer to take the more robust approach here.